addons/im_livechat/controllers/main.py

```diff
@@ -190,7 +190,7 @@ def get_session(self, channel_id, anonymous_name, previous_operator_id=None, cha
                 'name': channel_vals['name'],
                 'chatbot_current_step_id': channel_vals['chatbot_current_step_id'],
                 'state': 'open',
-                'operator_pid': (operator_partner.id, display_name.replace(',', '')),
+                'operator_pid': {'name': display_name, **operator_partner.mail_partner_format(fields={'id': True, 'write_date': True})[operator_partner]},
```

Comment:
`operator_pid` => `operator`

That way it matches the exact relational field in JS model